packages/iot-app-kit-visualizations/src/components/charts/sc-tooltip/sc-tooltip.tsx

```diff
@@ -55,6 +83,116 @@ export class ScTooltip {
     this.dataContainer.removeEventListener('mousedown', this.hideTooltip);
   }
 
+  tooltipHeight = (numRows: number) => numRows * TOOLTIP_ROW_HEIGHT + TOOLTIP_EMPTY_HEIGHT;
+
+  visualizedDataStreams = (): DataStream[] => {
```

Comment:
most changes here were functions i had to move over from `sc-tooltip-rows.tsx` to get positioning working :)